source/appModules/skypeapp.py

```diff
@@ -0,0 +1,160 @@
+#appModules/skypeapp.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2018 NV Access Limited, Joseph Lee
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+"""Routines for universal app version of Skype."""
+
+import re
+import appModuleHandler
+import ui
+from NVDAObjects.UIA import UIA
+from nvdaBuiltin.appModules.skype import SCRCAT_SKYPE
+import api
+import addonHandler
+addonHandler.initTranslation()
+
+# #7126: Unfortunately, cherry-picking parts of message items via looking at their children isn't reliable due to odd UIA implementation.
+# Therefore, resort to this regular expression.
+RE_MESSAGE = re.compile(r"\AFrom (?P<from>.*), Skype (?P<body>.*), sent on (?P<time>.*?)(?: Edited by .* at .*?)?(?: Not delivered|New)?\Z", re.M|re.S)
```

Comment:
This will only work for the English version of Skype UWP. Is the UIA implementation that unreliable that it justifies using a regex containing locale specific things?